sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -14,38 +14,14 @@ export const DATE_ONLY_ISO_FORMAT = "YYYY-MM-DD";
  */
 
 export function useFormatters() {
-  /**
-   * Get the date only part of a date/time or string.
-   * @param date date/time or string to have the date extracted.
-   * @returns date only string in ISO format YYYY-MM-DD.
-   */
-  const getISODateOnlyString = (date: Date | string): string => {
```

Comment:
`getISODateOnlyString` should not be removed. It should not be used to display dates on UI but is can be used to send date-only values to the API.

Reply:
I will put it back.

Reply:
I have put it back.